packages/expo-cli/src/askUser.ts

```diff
@@ -26,7 +25,7 @@ export default {
 };
 
 if (require.main === module) {
-  askForSendToAsync().then(function (sendTo) {
+  askForSendToAsync().then(function(sendTo) {
     console.log('Your email address is', sendTo);
   });
 }
```

Comment:
Maybe remove this part? It seems to enable something like running `node ./build/askUser.js` to manually test this module, but I don't believe anyone is actually doing this 😅